problem_builder/public/js/lms_util.js

```diff
@@ -0,0 +1,120 @@
+(function () {
+
+    /**
+     *  Manager for HTML XBlocks. These blocks are hid by detaching and shown
+     *  by re-attaching them to the DOM.This is only generic way to generically
+     *  handle things like video players (they should stop playing when removed from DOM).
```

Comment:
@jbzdak "hid" ⟶ "hidden"

"DOM.This is only generic way to generically handle" ⟶ "DOM. This is the only way to generically handle"